From 6cd0753cb003aa94b7b6329094d46bad5863fe68 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Wed, 29 Mar 2023 01:42:02 +0300
Subject: [PATCH 3/3] recovery: Force USB data_role to device

Some drivers set them to 'host' by default (observed on
PinePhonePro @ 6.3). Force it to 'device' so we can use fastbootd.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 common/base/init.recovery.glodroid.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/base/init.recovery.glodroid.rc b/common/base/init.recovery.glodroid.rc
index 6f310df..03274b7 100644
--- a/common/base/init.recovery.glodroid.rc
+++ b/common/base/init.recovery.glodroid.rc
@@ -1,4 +1,5 @@
 on init
+    setprop sys.usb.typec.data_role device
     setprop sys.usb.configfs 1
 
 service console /system/bin/sh
-- 
2.34.1

